src/main/java/org/truffleruby/interop/OutgoingForeignCallNode.java

```diff
@@ -296,6 +389,7 @@ protected static int expectedArity(String name) {
             case TO_A:
             case TO_ARY:
             case SIZE:
+            case LENGTH:
```

Comment:
This and below need to include the other new methods too, for a proper error when given an incorrect number of arguments.